CHANGELOG.md

```diff
@@ -50,6 +50,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - **CUMULUS-2638**
   - Transparent to users, remove typescript type `BucketType`.
```

Comment:
Can you also add a `Notable changes`? So DAAC knows to update their `task_config` for `SyncGranule`?

Reply:
If this sort of update is required, we should consider updating the doc here, or at least referring to this change somehow, else I can imagine some user confusion occurring. 

https://nasa.github.io/cumulus/docs/upgrade-notes/update-task-file-schemas

Reply:
Sorry, this was wrong, I should have said 'under the migration header'

Reply:
added a line under migration steps.